### ISSUE TITLE
facing "nan" values during training the model

### ISSUE THREAD

**Issue by baogiadoan** [locuslab/fast_adversarial#18]
hi, during the training with my custom objective loss, I realized that sometimes the model went wrong and produce "nan" and become invalid; which I didn't face before with other training methods, is that because of the *learning rate* of the cyclic learning rate being too large and causing the loss to diverge as mentioned in the paper:  `For each method, we individually tune λ to be as large as possible without causing the training loss to diverge`? or is it a bug? 

I ran the original again with `epochs=30` and also faced the same issue: 
![image](https://user-images.githubusercontent.com/4952756/113829398-9c80c100-97c4-11eb-94af-14a0f5300ea7.png)

**Comment by leslierice1** [comment]
You could try training with a smaller learning rate or clipping the (unscaled) gradients. Do you notice the same behavior with and without training with mixed precision?
